### ISSUE TITLE
Editable CSS

### ISSUE THREAD

**Issue by rpringle** [codeforamerica/city-analytics-dashboard-setup#6]
Would like a way to easily edit the CSS for the dashboard to make it easier to embed and blend with our current CMS styles.

**Comment by phae** [comment]
The current version isn't really intended to be something you'd embed.  Is this a use-case you have right now?  Would like to understand more about how you'd rather be using something like this.

**Comment by rpringle** [comment]
We intend to display it internally in at least IT on monitors in public spaces but I would also like to make it available from within the admin section of our CMS. This would give our content managers a better sense of who's using the site, search phrases, etc. directly in the place they spend most of their day.

Honestly, I'll probably just write my own module in PHP specifically for our CMS using the realtime API. I just thought it would be a quick win and a stop gap if I could deploy this dashboard in the meantime.

**Comment by phae** [comment]
Ah! I see. Funny, that's actually an idea we're playing with right now over in our other repo ceviche-cms - having inline analytics in the CMS.  For this version of the dashboard, though, we probably won't be making this a feature - you can just fork it and edit it as you need.  

I think in the long-run, I'd like to make something more configurable that can be used as you described, and is also the same thing that powers the analytics portions of our CMS.

**Comment by rpringle** [comment]
Makes sense. I gave Cyd and Jack feedback on ceviche-cms when they demo'd it a few weeks back. I really like the idea of inline analytics. I suspect for my needs I'll rebuild the dashboard in PHP but I've forked the project too. Thanks!
